.github/ISSUE_TEMPLATE/intent-to-ship--i2s-.md

```diff
@@ -51,8 +51,66 @@ Provide instructions for how to demo your feature such as a link to a demo page.
 Add any other information that may be relevant in determining if your feature can ship.
 -->
 
+## Build checklist
+
+This section describes considerations the implementor should keep in mind while building this technical solution.
+
+- [ ] Tests written
+<!--
+Have unit tests been written? Integration tests? Validator tests?
+Details on unit and integration tests:https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#testing-your-changes
+Details on validator tests: https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md
+-->
+- [ ] Documentation present
+<!--
+Has reference documentation been added? Have examples been added? Has an example been made available on amp.dev?
+-->
+- [ ] Cross platform testing
+<!--
+Have you tested all browsers and versions that AMP supports? https://amp.dev/support/faq/supported-browsers/
+Have you tested across mobile and desktop devices? 
+-->
+
+### Considerations checklist
+
+This section describes different cross cutting considerations that the implementor should keep in mind while building this technical solution.
+
+<!--
```

Comment:
Remove since sentence.

Reply:
I don't understand.

Reply:
Sorry, sentence jumbled. I meant to just remove this paragraph since we no longer want to remove any consideration.